web-components/docs/README.md

```diff
@@ -443,21 +454,21 @@ Show a ranking of US States by population, highest to lowest
 
 ```html
 <datacommons-ranking
-    title="US States with the Highest Population"
-    place="country/USA"
-    childPlaceType="State"
-    variable="Count_Person"
+  title="US States with the Highest Population"
+  parentPlace="country/USA"
+  childPlaceType="State"
+  variable="Count_Person"
 ></datacommons-ranking>
 ```
 
 Show a ranking of US States by population, lowest to highest
 
 ```html
 <datacommons-ranking
-    title="US States with the Lowest Population"
-    place="country/USA"
-    childPlaceType="State"
-    variable="Count_Person"
-    showLowest=true
+  title="US States with the Lowest Population"
+  parentPlace="country/USA"
+  childPlaceType="State"
+  variable="Count_Person"
+  showLowest="true"
```

Comment:
Note for later: We should get the ranking component to also allow just `showLowest`, without having to set `="true"`

Reply:
Good catch! I went ahead and added a fix for this in this PR